frontend/pages/ManageControlsPage/MacOSSettings/MacOSSettings.tsx

```diff
@@ -58,8 +81,9 @@ const MacOSSettings = ({ params, location }: IMacOSSettingsProps) => {
             key={team_id}
             currentTeamId={teamId}
             profiles={profiles}
-            onProfileUpload={refectchProfiles}
-            onProfileDelete={refectchProfiles}
```

Comment:
Leaving these arguments as refetchProfiles because that describes best what's happening